src/app/story/storyrow/totaltimes/totaltimes.component.css

```diff
@@ -0,0 +1,27 @@
+@media (min-width: 992px) {
```

Comment:
This media query needs to be after the `display: none` or else the `display: none` always applies.